README.md

```diff
@@ -22,12 +26,89 @@ $ head dependencies.nix
 
 You can then use this to download all the necessary dependencies to run your application.
 
+### Building a Maven repository
+
+Now that you have a **nix** dependencies file; we can re-construct a Maven repository using Nix!
+
+```nix
+let pkgs = import <nixpkgs> {}
+    buildMavenRepsitory = import (
+        fetchTarball https://github.com/fzakaria/mvn2nix/archive/master.tar.gz
+        ).buildMavenRepsitory;
```

Comment:
Looks like this is missing an invocation as `default.nix` evaluates to a function. So to make the above work as intended it would have to be:

```
  buildMavenRepsitory = (import (
    fetchTarball https://github.com/fzakaria/mvn2nix/archive/master.tar.gz
  ) { }).buildMavenRepsitory;
 ```

Maybe a bit of a mouth full ... how about this instead:

```nix
let mvn2nix = import (fetchTarball https://github.com/fzakaria/mvn2nix/archive/master.tar.gz) { };
in
mvn2nix.buildMavenRepsitory {}
```

Note that I also removed the unnecessary `pkgs` binding.